### ISSUE TITLE
enterEntity is broken.

### ISSUE THREAD

**Issue by AleziaKurdis** [kasenvr/project-athena#845]
enterEntity and leaveEntity seems broken. Those event fails to trigger (It was working in the paste)

STEPS TO REPRODUCE:
1- Display the Debug Window (**Developer > Scripting > Script Log (HMD friendly...)**)

2- Create a **Shape (Box**) entity

3- Resize it to be enough tall for your avatar to be able to fit inside of it. 
(but keep the box in front of you, you must not be inside at this moment.)

4- Set it collisionless

5- Add this script (Client Entity Script): 
_http://metaverse.bashora.com/scripts/curling/test_enterEntity.js_
This a very simple script:
################################
(function(){ 
    this.preload = function(entityID) {
        print("ACTIVATED!");
    }
    this.enterEntity = function(entityID) {
        print("ENTERED!");
    }; 
    this.leaveEntity = function(entityID) {
        print("LEFT!");
    };
})
#############################

5- Note that you can saw "**ACTIVATED!**" displayed in the Debug Window as the script loaded.

6- Enter the box entity...
Nothing happens. Normally , **enterEntity** would have printed "**ENTERED!**" in the Debug Window.  
(It works sometimes, if you have already visited the domain before, it might decide to work. 
Sadly, it's too bad for the newcomer. 

7- Exit the box... Nothing happens there too. 

I get this situation in different domains and serverless domain.

**Comment by ctrlaltdavid** [comment]
Just tested 2020.3.1 on localhost and it's working OK.
I entered the URL given in the "Script" entity property.
"LEFT!" and "ENTERED!" are being reported to the program log and script debug window, OK.
Also tested in the Community Village and the Help > Tutorial serverless domain.

It doesn't work if the script is loaded as a server entity script, though.
Should it?

**Comment by AleziaKurdis** [comment]
I reproduce it here:
http://metaverse.bashora.com/explore/serverlessEnterEntityTester.json

The first time I went on this serverless, the script worked.
I could see "ENTERED!" and "LEFT!" logged in the debug window when I crossed the yellow cube.
Then I did Reload Content, and the scripts get loaded (I could see "ACTIVATED!" printed in the debug window)
but nothing happens when I crosse the yellow cube.
So I re teleport to this location (using explore app, by pasting the URL)
the script didn't work more after.
Then I quit Interface, and relaunched it, then went to that location
then the script was working fine.

So to recap, after have done "Reload Content", enterEntity stops to work until we restart interface.

**Comment by AleziaKurdis** [comment]
Tested on 
PR #853 Fix enterEntity event not firing in entity script after content reload
with success.
